crates/ika-core/src/dwallet_mpc/dwallet_mpc_service.rs

```diff
@@ -97,51 +100,57 @@ impl DWalletMPCService {
         }
     }
 
-    /// Load missed events from the Sui network.
-    /// These events are from different Epochs, not necessarily the current one.
-    ///
-    async fn load_missed_events(&mut self) {
+    /// Proactively pull uncompleted events from the Sui network.
+    /// We do that to assure we don't miss any events.
+    /// These events might be from a different Epoch, not necessarily the current one.
+    async fn fetch_uncompleted_events(&mut self) -> Vec<DWalletMPCEvent> {
```

Comment:
Returning a `Vec<>` now, because I want to have unified handling to both uncompleted and new events